router/internal/timex/duration.go

```diff
@@ -0,0 +1,22 @@
+package timex
+
+import (
+	"math/rand"
```

Comment:
You probably rather want to use `math/rand/v2`
https://pkg.go.dev/math/rand/v2

Otherwise you need to seed

Reply:
thanks